Sample Applications/WPFGallery/Views/SettingsPage.xaml

```diff
@@ -17,32 +46,138 @@
             Grid.Row="0"
             Margin="0,0,0,40"
             Title="{Binding ViewModel.PageTitle}"
-            Description="{Binding ViewModel.PageDescription}"
-            ShowDescription="False" />
-
-        <StackPanel Margin="10 0 10 24" Grid.Row="1">
-            <TextBlock Margin="10 0 0 16" FontSize="24" FontWeight="SemiBold" Text="Appearance &amp; behavior" />
-            <Grid Margin="10 0 10 20">
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="Auto" />
-                    <ColumnDefinition Width="*" />
-                    <ColumnDefinition Width="Auto" />
-                </Grid.ColumnDefinitions>
-                <TextBlock Grid.Column="0"  FontSize="28" Padding="12" FontWeight="SemiBold">
-                    <!--<SymbolIcon Symbol="Color24" />-->
-                    
-                </TextBlock>
-                <StackPanel Orientation="Vertical" Grid.Column="1" VerticalAlignment="Center">
-                    <TextBlock FontSize="18" FontWeight="SemiBold" Text="System Theme"/>
-                    <TextBlock Text="Select which app theme to use" />
-                </StackPanel>
-                <!--<ComboBox Grid.Column="2" MinWidth="200" SelectedIndex="{Binding ViewModel.CurrentApplicationTheme, Converter={StaticResource ThemeToIndexConverter}, Mode=TwoWay}">
-                    <ComboBoxItem Content="Light" />
-                    <ComboBoxItem Content="Dark" />
-                    <ComboBoxItem Content="High Contrast" />
-                </ComboBox>-->
+            Description="{Binding ViewModel.PageDescription}"/>
+
+        <StackPanel Grid.Row="1" Margin="5,5,5,24">
+            <TextBlock Foreground="{DynamicResource TextFillColorPrimaryBrush}" Text="Appearance &amp; behavior" FontWeight="SemiBold" Margin="10" FontSize="14"/>
```

Comment:
We don't need to set Foreground="{DynamicResource TextFillColorPrimaryBrush}" for the text block's separately.
I guess, they will be handled by line 12.